### PR TITLE
[12.0][IMP] sales_team_operating_unit: Improved unit test

### PR DESCRIPTION
**Pull request by murtuzasaleh**

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
#306 It will solve UT Case.

File: sales_team_operating_unit/tests/test_crm_team_operating_unit.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ def _create_user(self, login, groups, company, operating_units,
             'email': 'test@yourcompany.com',
             'company_id': company.id,
             'company_ids': [(4, company.id)],
+            'default_operating_unit_id': self.ou1.id,
```
Comment (AaronHForgeFlow) [inline sales_team_operating_unit/tests/test_crm_team_operating_unit.py]: I think it is more logical to assign the first operating unit passed to that method: operating_units[0].id

**Review by max3903 — APPROVED** [review]

**Review by AaronHForgeFlow — COMMENTED** [review]
why the previous fix failed? current fix is changing the main purpose of the tests for sale_team_operating_unit. Adding user 2 to both operating units is "skipping" the checks.

**Comment by murtuzasaleh** [comment]
> why the previous fix failed? current fix is changing the main purpose of the tests for sale_team_operating_unit. Adding user 2 to both operating units is "skipping" the checks.

@AaronHForgeFlow  Done, Can you please review it again?

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:  LGTM Could you squash commits before merging?

**Comment by murtuzasaleh** [comment]
>  LGTM Could you squash commits before merging?

Done

**Comment by AaronHForgeFlow** [comment]
> Done

Thanks.

/ocabot merge patch

**Pull request merged**